walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -28,6 +29,16 @@
   private final TimeRange when;
   private final Set<String> attendees = new HashSet<>();
 
+  /**
+  * A comparator for sorting events by their start time in ascending order.
```

Comment:
Align the *  (similar to line 24).  Here and in the rest of the files.

Reply:
Addressed: 80d9acb